core.js

```diff
@@ -0,0 +1,40 @@
+import AggregateError from 'aggregate-error'; // Use built-in when targeting Node.js 16
+
+export class IpNotFoundError extends Error {
+	constructor(options) {
+		super('Could not get the public IP address', options);
+		this.name = 'IpNotFoundError';
+	}
+}
+
+export function createPublicIp(publicIpv4, publicIpv6) {
+	return function publicIp(options) { // eslint-disable-line func-names
```

Comment:
The function is named to improve error messages but xo doesn't allow it to be immediately exported.

![image](https://user-images.githubusercontent.com/29491356/169220648-bef6d458-fb8d-496f-9519-910339df4b06.png)